app/views/templates/search/search_object.rb

```diff
@@ -151,8 +155,19 @@ def content
                 }
               ]
             }),
-
-            description: render_document_show_field_value(document, 'proxies.dcDescription'),
+            description: data_section( {
+               :title => 'site.object.meta-label.description',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.